js/privatebin.js

```diff
@@ -1195,8 +1195,17 @@ jQuery.PrivateBin = (function($, sjcl, Base64, RawDeflate) {
          */
         function sendToShortener()
         {
-            window.location.href = $shortenButton.data('shortener') +
-                                   encodeURIComponent($pasteUrl.attr('href'));
+            var url = $shortenButton.data('shortener')+encodeURIComponent($pasteUrl.attr('href'));
+
+            if($shortenButton.data('ajax'))
```

Comment:
Please add the brackets for the if block to improve readability, also for the else block.